packages/atomic/src/components/common/breadbox/breadbox.tsx

```diff
@@ -0,0 +1,241 @@
+import {FacetValueState} from '@coveo/headless';
```

Comment:
For the sake of uniformity, if you compare with other recently refactor functional components, they are all put into separate files that matches the component name.

We should have the same structure here.

```
common/breadcrumb/container.tsx
     export const BreadcrumbContainer: FunctionalComponent<...>

common/breadcrumb/showless.tsx
     export const BreadcrumbShowLess: FunctionalComponent<...>
```

It also makes it so that for a new use case, we could pick and chose "only some" functional components, and replace some other by more specific ones. By putting them all in the same files, its less obvious.